src/Engines/constants.ts

```diff
@@ -164,6 +164,8 @@ export class Constants {
   public static readonly TEXTUREFORMAT_DEPTH32_FLOAT = 14;
   /** Depth 16 bits */
   public static readonly TEXTUREFORMAT_DEPTH16 = 15;
+  /** Depth 24 bits */
+  public static readonly TEXTUREFORMAT_DEPTH24 = 16;

```

Comment:
You need to also update the file components/tabs/propertyGrids/materials/texturePropertyGridComponent.tsx of the inspector to add support for this new type of format (see around line 63).